doc/modules/clustering.rst

```diff
@@ -392,22 +392,22 @@ for centroids to be the mean of the points within a given region. These
 candidates are then filtered in a post-processing stage to eliminate
 near-duplicates to form the final set of centroids.
 
-Given a candidate centroid :math:`x_i` for iteration :math:`t`, the candidate
+Given a candidate centroid :math:`x` for iteration :math:`t`, the candidate
 is updated according to the following equation:
 
 .. math::
 
-    x_i^{t+1} = m(x_i^t)
+    x^{t+1} = x^t + m(x^t)
 
-Where :math:`N(x_i)` is the neighborhood of samples within a given distance
-around :math:`x_i` and :math:`m` is the *mean shift* vector that is computed for each
+Where :math:`N(x)` is the neighborhood of samples within a given distance
+around :math:`x` and :math:`m` is the *mean shift* vector that is computed for each
 centroid that points towards a region of the maximum increase in the density of points.
 This is computed using the following equation, effectively updating a centroid
 to be the mean of the samples within its neighborhood:
 
 .. math::
 
-    m(x_i) = \frac{\sum_{x_j \in N(x_i)}K(x_j - x_i)x_j}{\sum_{x_j \in N(x_i)}K(x_j - x_i)}
+    m(x)  = \frac{1}{|N(x)|} \sum_{x_j \in N(x)}x_j - x
```

Comment:
I think it could still be worth to have a line making appear the kernel `K` and define it with the way we used it in scikit-learn.

Reply:
Done